src/app/settings/page.tsx

```diff
@@ -53,6 +56,8 @@ export default function ProfilePage() {
       phoneNumber: '',
       gender: '',
       industry: '',
+      jobTitle: '',
+      company: '',
```

Comment:
![image](https://github.com/user-attachments/assets/015753ca-f792-4209-a1a8-851176e0ee53)

Reply:
Occurs, when no phone number is submitted in the form

Reply:
word i'll make sure all fields are optional to update